languages/javascript/exercises/concept/arrays-basic/.docs/introduction.md

```diff
@@ -0,0 +1,8 @@
+In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or ojects, even mixed types. The array elements can be accesed by their index.
+
+Examples of equivalent declaration:
+
+```javascript
+const numbers = [1, 'two', 3, 'four']; // most common`
+const numbers = new Array(1, 'two', 3, 'four');
```

Comment:
I think this was meant to move to `after.md` and only showing the `[]` syntax.

Reply:
Yep, it was.

Reply:
I think I messed up the commits. I added back the missing changes, so it should all be good now. Maybe my Git skills need some brush up.